Documentation/general/packaged-shim.md

```diff
@@ -0,0 +1,55 @@
+Packaged shim
+===============================
+
+For DotnetTools, if signed shim is required. You need to add the following extra property in you project file. Then run `dotnet pack`. After the nupkg is generated, shim is generated in the package. You can signed all the content of the nupkg which includes the shims.
+
+During `dotnet tool install` that shim is used instead of creating one on consumer's machine.
```

Comment:
During `dotnet tool install` if a shim is found in the nupkg, it is used instead of creating one on consumer's machine.